### DISCUSSION TITLE
升级版本后，DNS泄漏

### DISCUSSION THREAD

**Discussion by careysucci** [immortalwrt/homeproxy#211] [Q&A]
1、10.16从kenzo仓库拉homeproxy打包，会出现ip与dns泄漏，查面板ip泄漏匹配到了直连规则集
于是重新打包改成拉该仓库打包homeproxy，出现了如题问题
2、**ip不泄露了**，werbrtc也不泄露，但是dns还是泄漏，默认dns是外网dns，解析策略是：仅ipv4
请问怎么排查或者怎么解决该问题呀？

**Comment by 1715173329** [comment]
去买检测泄漏网站的 VPN 即可，250% 安全。

**Reply by careysucci** [reply]
是说他会帮你弄好，还是他的网站不会显示他vpn会泄漏？
用的是这个网站检测的：https://browserleaks.com/ip

**Reply by mqcycy** [reply]
> 是说他会帮你弄好，还是他的网站不会显示他vpn会泄漏？
> 用的是这个网站检测的：https://browserleaks.com/ip

🤭逗你玩的，
